edb/server/dbview/dbview.pyx

```diff
@@ -1303,7 +1303,7 @@ cdef class DatabaseIndex:
             return self._dbs[dbname]
         except KeyError:
             raise errors.UnknownDatabaseError(
-                f'database {dbname!r} does not exist')
+                f'database branch {dbname!r} does not exist')
```

Comment:
Keeping `database` here seems debatable, but I don't have a strong argument in favor of either direction, so lgtm!

Reply:
We've done similar in a lot of places in the server